### DISCUSSION TITLE
Can we infer failure metrics from an orphaned start event?

### DISCUSSION THREAD

**Discussion by aaronsteers** [meltano/meltano#6040] [General]
@pnadolny13, @WillDaSilva, @pandemicsyn, @tayloramurphy 

From the related discussion, and based on the complexity of putting a failed event everywhere, I wanted to start this thread to see if we feel okay _not_ having a failure event in all cases.

The logic would presumable be:

- Given a specific settle period like 24 or 48 hours, we would infer a failure if no completed event was received for the specific start event.

**Comment by pnadolny13** [comment]
@aaronsteers Ideally we'd do our best to capture as many failed/aborted events as possible but where its not possible we'll fall back on inferring failures. Obviously though I understand the complexity so covering as much as reasonable.

I think we'll need to infer failure/abort either way because we wont be able to fire events on hard exits. Personally I do this a lot, I'm executing something then as it starts I realize my credentials arent set or the configs werent correct so I ctrl+C hard exit. That case is mostly for local executions but killing a K8s container or Airflow task probably acts the same so we'll still need to handle inferring in the transformations either way. We would definitely want to try to capture the case where a long running job gets manually killed by the user or it reaches their timeout setting which we cant get from inferring.
